.circleci/config.yml

```diff
@@ -303,6 +303,12 @@ jobs:
           paths:
             - cc-test-reporter
             - coverage
+      - run:
+          name: Copy CMS log
+          command: cp integreat_cms/integreat-cms.log integreat-cms.log
```

Comment:
This intermediate copying is necessary to apply the on_fail condition. Because the store_artifacts step does not support conditions.